sample/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>CKEditor 4 – Vue.js Component – development sample</title>
+	<style>
+		body {
+			max-width: 800px;
+			margin: 20px auto;
+		}
+
+		.cke_editable_inline {
+			border: solid 1px black;
+		}
+
+		.highlight {
+			border-radius: 2px;
+			background: lightgray;
+			border: solid 1px darkgray;
+			padding-left: 4px;
+			padding-right: 4px;
+			color: darkred;
+		}
+
+		.route__list {
+			list-style-type: none;
+		}
+	</style>
+</head>
+<body>
+	<script src="../node_modules/vue/dist/vue.js"></script>
+	<script src="../node_modules/vue-router/dist/vue-router.js"></script>
+	<script src="../dist/ckeditor.js"></script>
+
+	<div id="app">
+		<h1>CKEditor 4 – Vue.js Component – development sample</h1>
+
+		<ul class="route__list">
+		<li class="route__list-item"><router-link to="/types">Editor types</router-link></li>
+		<li class="route__list-item"><router-link to="/events">Component events</router-link></li>
+		<li class="route__list-item"><router-link to="/binding">Two way binding</router-link></li>
+		</ul>
+
+		<router-view></router-view>
+	</div>
+
+	<script type="text/x-template" id="types">
+		<main>
+			<h2>Editor types</h2>
+
+			<section>
+				<h3>Classic Editor</h3>
+				<ckeditor
+					ref="classic"
+					type="classic"
+					tag-name="div"
+					v-model="editors.classic.data"
```

Comment:
Seems like could be cut into `editors.classic` (of course, requires model change).